ibis/backends/dask/execution/selection.py

```diff
@@ -137,6 +141,7 @@ def execute_selection_dataframe(
 
     # Build up the individual dask structures from column expressions
     if selections:
+        data, _ = add_partitioned_sorted_column(data, set_index=True)
```

Comment:
What is the `_` here and why do we not need it?

Reply:
Can you explain how this works in the comment? i.e., what uses partition column and what's the effect of that.

Reply:
I think the docstring for the function covers both of these questions.

Reply:
As a reader this code is not obvious to me. Actually I only understood this after reading through the whole review for a second time. So it would be helpful for future reader explaining what is going on here.

Reply:
Apologies - added clarifying comments that hopefully make this easier to follow.

Reply:
Thanks for adding this!